tools/bridge/tests/test_event_fetcher.py

```diff
@@ -271,20 +261,44 @@ def test_fetch_events_not_seen_handle_event_limit_not_exact_multiplicate(
     assert transfer_event_queue.qsize() == transfer_count
 
 
+def test_fetch_events_not_seen_apply_start_block_number(
+    transfer_event_fetcher,
+    w3_foreign,
+    tester_foreign,
+    transfer_event_queue,
+    transfer_tokens_to_foreign_bridge,
+    foreign_chain_max_reorg_depth,
+):
+    assert transfer_event_queue.empty()
+
+    transfer_tokens_to_foreign_bridge()
+    transfer_event_fetcher.last_fetched_block_number = w3_foreign.eth.blockNumber
+    transfer_tokens_to_foreign_bridge()
+    tester_foreign.mine_blocks(foreign_chain_max_reorg_depth)
+    transfer_event_fetcher.fetch_events_not_seen()
+
+    assert transfer_event_queue.qsize() == 1
+
+
 def test_fetch_events_continuously(
-    transfer_event_fetcher, transfer_event_queue, transfer_tokens_to_foreign_bridge
+    transfer_event_fetcher_init_kwargs,
+    tester_foreign,
+    transfer_event_queue,
+    transfer_tokens_to_foreign_bridge,
 ):
     assert transfer_event_queue.empty()
 
     poll_time = 1
-    transfer_event_fetcher.max_reorg_depth = 0
+    transfer_event_fetcher = EventFetcher(
+        **{**transfer_event_fetcher_init_kwargs, "max_reorg_depth": 0}
+    )
 
     try:
         greenlet = Greenlet.spawn(transfer_event_fetcher.fetch_events, poll_time)
         transfer_tokens_to_foreign_bridge()
         sleep(poll_time + 1)
 
-        assert transfer_event_queue.qsize() == 1
+        #  assert transfer_event_queue.qsize() == 1
```

Comment:
Why is this outcommented?

Reply:
Because I'm silly and haven't read again my changes until the last line on too many changes. Will commented out of cause and the test case still succeed.

Reply:
:heavy_check_mark: